db/migrations/20220408204736-create-paybuttons.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('paybuttons', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.STRING,
```

Comment:
If this is a userID it should probably be an integer. If its a username though, lets rename it

Reply:
In this case it should be a string as it refers to a user_id on the supertokens server: https://github.com/supertokens/core-driver-interface/wiki

Reply:
we could rename it to supertokens_user_id or user_from_provider_id or something more explicit, do you have any suggestion?

Reply:
Yeah lets rename that to either of `SupertokensUserId`, or `ProviderUserId` is fine

Reply:
done